content/updates/2020-07/2-rules-date-format.md

```diff
@@ -0,0 +1,19 @@
+---
+pim_announcement_audience:
+- EE
+---
+
+# A new relative date format
+
+This is now possible to filter products/product models based on a **relative date** in the rules engine. This new format is available via YML import only for now, and for these 3 filters: `updated`, `created`, and `date` attribute type.
```

Comment:
```suggestion
You can now create a rule based on a **relative date** condition. This new condition can be currently used only through YML import (it will be shortly available in our brand new rules engine screen!) and for the `updated` and `created` system attributes and the `date` attribute type.
```